platform/ios/MGLCategoryLoader.m

```diff
@@ -4,7 +4,16 @@
 #import "NSProcessInfo+MGLAdditions.h"
 #import "NSString+MGLAdditions.h"
 
+#import "MGLMapView.h"
+
 // https://github.com/mapbox/mapbox-gl-native/issues/2966
+// https://github.com/mapbox/mapbox-gl-native/issues/3113
+
+@interface MGLMapView (MGLCategoryLoader)
+
++ (void)mgl_linkMapView;
```

Comment:
Come to think of it, ObjC dynamic message dispatch would probably prevent any optimization around whatever class method you’d call here, but I feel better about a method that has to return a string than one that really is a no-op.